terraform/main.tf

```diff
@@ -32,9 +32,35 @@ data "aws_ecs_cluster" "c13-cluster" {
   cluster_name = var.CLUSTER_NAME
 }
 
+# This locals defined the environment variables needed by our
+# Lambdas and ECS dashboard service. In the variables.tf file 
+# a sensitive tag is sometimes applied to ensure the values themselves
+# don't appear in terminal logs, or ouputs of any kind. Truly secure!
+locals {
+  common_env_vars = {
+    # Database connection variables
+    DB_HOST        		= var.DB_HOST
+	# In case you Lambda want to refer to ports.
+	# They are hardcoded here, as they are not sensitive.
+	DB_PORT				= "5432"
+	DASHBOARD_PORT 		= "8501"
+    DB_NAME             = var.DB_NAME
+    DB_USER             = var.DB_USER
+    DB_PASSWORD         = var.DB_PASSWORD
+
+    # API Keys needed for Lambdas running pipelines.
+    NASA_API_KEY = var.NASA_API_KEY
+    ASTRONOMY_KEY_ID  = var.ASTRONOMY_KEY_ID
+	ASTRONOMY_SECRET = var.ASTRONOMY_SECRET
+		
+
+	# More to be added when other pipeline come online..
+  }
+}
+
```

Comment:
Really like this function

Reply:
Agree with the decision to include all environment variables in the terraform.

Reply:
I agree with the agreement